viewer/packages/cad-geometry-loaders/src/sector/culling/TakenV8SectorTree.test.ts

```diff
@@ -90,35 +90,35 @@ describe('TakenSectorTree', () => {
 
   test('toWantedSectors includes provided geometryFilterBox', () => {
     const box = new THREE.Box3();
-    const root = generateSectorTree(2);
-    const tree = new TakenSectorTree(root, determineSectorCost);
+    const root = generateV8SectorTree(2);
+    const tree = new TakenV8SectorTree(root, determineSectorCost);
     const wanted = tree.toWantedSectors(model.modelIdentifier, model.modelBaseUrl, box);
 
     wanted.forEach(x => expect(x.geometryClipBox).toBe(box));
   });
 
   test('toWantedSectors only returns actual sectors when sectorIds are "sparse"', () => {
-    const root = generateSectorTree(3, 2);
+    const root = generateV8SectorTree(3, 2) as SectorMetadata;
```

Comment:
```suggestion
    const root : SectorMetadata = generateV8SectorTree(3, 2);
```
I like this type of declaration better...

However, I think the test should be rewritten to use the "full type", which I believe just involves changing `Mutable<SectorMetadata>` to `Mutable<V8SectorMetadata>` below.

Reply:
wont-do due to WithChildren conflicting with derived type.